e2e/pageobjects/ide/Ide.ts

```diff
@@ -75,35 +101,69 @@ export class Ide {
         await this.driverHelper.waitVisibility(By.css(Ide.LEFT_CONTENT_PANEL_CSS));
     }
 
-    async waitPreloaderAbsent(attempts: number = TestConstants.TS_SELENIUM_DEFAULT_ATTEMPTS, polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING) {
+    async waitPreloaderAbsent(attempts: number = TestConstants.TS_SELENIUM_DEFAULT_ATTEMPTS,
+        polling: number = TestConstants.TS_SELENIUM_DEFAULT_POLLING) {
         await this.driverHelper.waitDisappearance(By.css(Ide.PRELOADER_CSS), attempts, polling);
     }
 
     async waitStatusBarContains(expectedText: string, timeout: number = TestConstants.TS_SELENIUM_LANGUAGE_SERVER_START_TIMEOUT) {
         const statusBarLocator: By = By.css('div[id=\'theia-statusBar\']');
 
-        await this.driverHelper.waitUntilTrue(async () => {
+        await this.driverHelper.getDriver().wait(async () => {
             const elementText: string = await this.driverHelper.waitAndGetText(statusBarLocator, timeout);
+            const isTextPresent: boolean = elementText.search(expectedText) > 0;
 
-            return elementText.search(expectedText) > 0;
+            if (isTextPresent) {
```

Comment:
Not exactly.
Method "wait" waits until some value is returned,
if we return "false" method will pass,
so that's why we need this "if" check and return 
value just in case of successful check

Reply:
```
For a {@link webdriver.Condition} or function, the wait will repeatedly evaluate the condition until it returns a truthy value. 
```
From this it seems to me, that returning `false` should cause the wait loop to be executed again...

Reply:
You can check it yourself, it works as I described

Reply:
Hehe... Unfortunately I dont have time to try now, so I believe you :-D